src/components/avatar/index.js

```diff
@@ -195,11 +194,11 @@ export default class Avatar extends BaseComponent {
     return (
       <Badge
         backgroundColor={badgeColor}
-        borderWidth={DEFAULT_BADGE_BORDER_WIDTH}
         borderColor={DEFAULT_BADGE_BORDER_COLOR}
         containerStyle={this.getBadgePosition()}
         size={this.getBadgeSize()}
         {...badgeProps}
```

Comment:
This will override all the props you're passing to Badge. Are you sure you want to allow that?

Reply:
Yup, No other way to allow control over the Avatar badge from the outside. props supplied before the badgeProps override serves as defaults.